src/general/OptionsMenu.tscn

```diff
@@ -971,6 +971,13 @@ margin_bottom = 381.0
 action_mode = 0
 text = "OPEN_SCREENSHOT_FOLDER"
 
+[node name="LogButton" type="Button" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
```

Comment:
I think this should be put in a same hbox as the open screenshot, as I think the text would fit that way, and less vertical space would be wasted for all the buttons. What do you think @Kasterisk ?

Reply:
Sorry had to do something earlier. Yeah, that makes sense, I'm fine with the decision.

Reply:
Good, as I just merged after making that change.